pkgs/development/interpreters/lua-5/hooks/luarocks-move-data.sh

```diff
@@ -0,0 +1,15 @@
+# luarocks installs data in a non-overridable location. Until a proper luarocks patch,
+# we move the files around ourselves
+echo "Sourcing luarocks-move-data-hook.sh"
+
+luarocksMoveDataHook () {
+    echo "Executing luarocksMoveDataHook"
+    if [ -d "$out/$rocksSubdir" ]; then
+        cp -rfv "$out/$rocksSubdir/$pname/$version/." "$out"
+    fi
+
+    echo "Finished executing luarocksMoveDataHook"
+}
+
+echo "Using luarocksMoveDataHook"
```

Comment:
Isn't that a bit much noise for just copying some files?

Reply:
this is one of the tricky part so I wanted it to stand out:
- there is no way to tell luarocks to install "data" in the root folder 
- it is easy to plug in only for vim plugins and makes it clear it only concerns these plugins